examples/pennylane_g_transformation.py

```diff
@@ -0,0 +1,132 @@
+r"""
+.. _g_transformation:
+
+Gaussian Transformation
+=======================
+
+This tutorial demonstrates the basic working principles of PennyLane for
+continuous variable (CV) photonic devices. For more details about
+photonic quantum computing, the `Strawberry
+Fields <https://strawberryfields.readthedocs.io/en/latest/>`__
+documentation is a great starting point.
+
+ **GOAL:** To optimize displacement gate parameters in order to displace
+ a *vacuum mode* in the phase space in order to get one *qumode*.
+
+The Quantum Circuit
+~~~~~~~~~~~~~~~~~~~
+
+For this basic example, we will consider a special subset of CV
+operations: *Gaussian transformations*. We work with the following
+simple Gaussian circuit:
+
+.. figure:: ../../examples/figures/gaussian_transformation.svg
+    :align: center
+    :width: 70%
+    :target: javascript:void(0);
+
+What is this circuit doing?
+
+1. **We begin with one wire (qumode) in the vacuum state.** Note that we
+   use the same notation :math:`|0\rangle` for the initial state as
+   in the qubit rotation example. In a photonic CV system, this state is
+   the vacuum state i.e., the average photon number in the wire is zero.
+2. **We displace the qumode.** The displacement gate linearly shifts the
+   state of the qumode in phase space. The vacuum state is centered at
+   the origin in phase space while the displaced state will be centered
+   at the point :math:`\alpha`.
+3. **We rotate the qumode.** This is another linear transformation in
+   phase space, albeit a rotation (by angle :math:`\phi`) instead of a
+   displacement.
+4. **Finally, we measure the mean photon number
+   :math:`\langle\hat{n}\rangle=\langle\hat{a}^{\dagger}\hat{a}\rangle`.**
```

Comment:
Something is wrong with the math rendering!

Reply:
ah yes! thanks! its the use of ** to make the text bold. it does not work on math expressions.